### DISCUSSION TITLE
Mania Key count in description window

### DISCUSSION THREAD

**Discussion by jsmith1140974** [ppy/osu#18325] [Ideas] [answered]
I was wondering if the mania key count # could be added to the description window which lists BPM and song length (etc.), instead of having to rely on it being listed in the song name or song description (which doesn't always happen).

**Comment by frenzibyte** [comment, marked as answer]
Mania key count is already displayed in the beatmap statistics in "details" section, replacing "Circle Size". It'll also always be displayed in the new design of song select, rather than being nested in tabs.

**Reply by jsmith1140974** [reply]
whelp, I am an idiot. THank you for confirming!
